data/yam/agama/auto/template.jsonnet

```diff
@@ -0,0 +1,12 @@
+local lib = import 'lib/base.libsonnet';
+
+function(storage='default', product='SLES', scripts='default', encrypted=false) {
```

Comment:
as I mentioned to you, the 'default' word is too tricky and it is already used by schema, so it difficult understanding, we could go just with `storage=''` if that is allow?